pkg/console/operator/operator.go

```diff
@@ -222,8 +221,8 @@ func (c *consoleOperator) handleSync(operatorConfig *operatorsv1.Console, consol
 
 // this may need to move to sync_v400 if versions ever have custom delete logic
 func (c *consoleOperator) removeConsole(cr *operatorsv1.Console) error {
-	logrus.Info("deleting console resources")
-	defer logrus.Info("finished deleting console resources")
+	klog.Info("deleting console resources")
```

Comment:
```suggestion
	klog.V(4).Info("deleting console resources")
```

Reply:
I'm going to put both of these at:

```
klog.V(2).Info("deleting console resources")
defer klog.V(2).Info("finished deleting console resources")
```
I think you may want to see this in normal log output as it would run if `ManagementState:Removed`.  It ought to be about the only thing that should be seen in the logs.